todo_app/templates/index.html

```diff
@@ -3,16 +3,52 @@
 
 {% block content %}
   <div class="jumbotron">
-    <h1 class="display-4">To-Do App</h1>
-    <p class="lead">Just another to-do app.</p>
+    <h1 class="display-4">JB's</h1>
+    <h2 class="display-5">To-Do App</h2>
+    <p class="lead">Just another to-do app!</p>
   </div>
 
-  <div class="row">
-    <div class="col-md-12">
-      <h2>Items</h2>
-      <ul class="list-group mb-4">
-        <!-- List the todo items here -->
-      </ul>
+  <div class="container">
+      <h2>To-Do List</h2>
+      <hr>
+      <form action="" method="POST" class="form-example">
+        <div class="form-group">
+           <input type="text" name="title" class="form-control" ng-model="todoInput" size="40" placeholder="What do you need to do today?">
+        </div>
+        <input type="submit" value="Add" class="btn btn-success">
+      </form>
+      <table class="table">
+        <ul class="list-group mb-4">
+          <thead>
+            <tr>
+              <th scope="col">Task Lists</th>
+              {% if request.values.get('sort') == None %}
+              <th  scope="col"><a href="/?sort=2" class="table">Status</a></th>
+              {% elif request.values.get('sort') == '2' %}
+              <th scope="col"><a href="/?sort=1" class="table">Status ↕</a></th>
```

Comment:
Would expect the arrow ↕ when it's not sorted and up/down arrows for the other two cases to show that it's sorted ascending/descending

Reply:
Amended arrows to reflect correct sort status